### PR TITLE
Pin minio to a version that doesn't break makemigrations

### PR DESCRIPTION
**Pull request by waxlamp**

### CODE REVIEW

**Review by marySalvi — APPROVED** [review]

**Review by annehaley — APPROVED** [review]
We did the same thing on MIQA

File: docker-compose.yml [hunk 1]
```diff
@@ -25,7 +25,7 @@ services:
   minio:
     logging:
       driver: none
-    image: bitnami/minio:2022.3.3
+    image: minio/minio:RELEASE.2022-03-03T21-21-16Z
```
Comment (annehaley) [inline docker-compose.yml:28]: Is there not a x.x.x tag for this release? If not, is this even a formal release that we should pin to?

Comment (waxlamp) [reply docker-compose.yml:28]: For better or worse, all of the [minio image release tags](https://github.com/minio/minio/tags) look like this. These are, AFAICT, the formal releases.

Comment (annehaley) [reply docker-compose.yml:28]: Ok, thanks for clarifying. This looks good then, consider my approval refreshed :grin:

**Review by marySalvi — APPROVED** [review]

**Pull request merged**